spirit/core/static/spirit/stylesheets/reed-base.css

```diff
@@ -123,14 +123,14 @@ body
 /* governs button styling across forum ... could be improved */
 .pagecontent {
   width: 100%;
-  padding: 3% 3%;
+  padding: 3% 3% 3% 3%;
   margin-top: 0px;
+  background: rgba(255, 255, 255, 0.3);
 }
 
 .header, .pagecontent {
   margin: auto;
   max-width: 68em;
-  background: white;
```

Comment:
Okay, now it's black instead of grey. Is that easier to read?
<img width="1177" alt="screen shot 2017-08-11 at 10 42 01 am" src="https://user-images.githubusercontent.com/23424276/29225089-c020883a-7e81-11e7-8e05-5dfaaf85e9ee.png">
<img width="812" alt="screen shot 2017-08-11 at 10 46 23 am" src="https://user-images.githubusercontent.com/23424276/29225235-5d5bc63c-7e82-11e7-8b93-facc80dc3673.png">